build.assets/Makefile

```diff
@@ -263,6 +264,10 @@ clean:
 .PHONY: publish-telekube
 publish-telekube: build
 	TELEPORT_REPOTAG=$(TELEKUBE_TSH_TAG) $(MAKE) build-tsh
+	if [ -z "$(TELE_KEY)" ] || [ -z "$(DISTRIBUTION_OPSCENTER)" ]; then \
+	   echo "TELE_KEY or DISTRIBUTION_OPSCENTER are not set"; exit 1; \
+	fi;
+	tele login --ops $(DISTRIBUTION_OPSCENTER) --key=$(TELE_KEY) $(TELEKUBE_PUBLISH_FLAGS)
```

Comment:
I'd prefer to do this without`/usr/bin/tele` so we can keep #1154  closed but I was unable to figure out how to properly do so.

I tried `$(TSH_OUT) login` but coudn't figure out the right combination of flags to mimic the `tele login` functionality.

Alternatively, I looked for `gravity package delete/import` `--hub` and `--token` flags similar to `tele push/pull`.  I didn't find them.  Should these be present?  If so, I can cut a ticket for it.